docs/src/main/sphinx/admin/preview-web-interface.md

```diff
@@ -30,3 +30,44 @@ Once activated, users can access the interface in the URL context `/ui/preview`
 after successful login to the [](/admin/web-interface). For example, the full
 URL on a locally running Trino installation or Trino docker container without
 TLS configuration is [http://localhost:8080/ui/preview](http://localhost:8080/ui/preview).
+
+## Authentication
+
+The Preview Web UI requires users to authenticate. If Trino is not configured to
+require authentication, then any username can be used, and no password is
+required or allowed. Typically, users login with the same username that they use
+for running queries.
+
+If no system access control is installed, then all users are able to view and
+kill any query. This can be restricted by using [query rules](query-rules) with
+the [](/security/built-in-system-access-control). Users always have permission
+to view or kill their own queries.
+
+### Password authentication
+
+Typically, a password-based authentication method such as [LDAP](/security/ldap)
+or [password file](/security/password-file) is used to secure both the Trino
+server and the Web UI. When the Trino server is configured to use a password
+authenticator, the Web UI authentication type is automatically set to `FORM`. In
+this case, the Web UI displays a login form that accepts a username and
+password.
```

Comment:
It might just be my experience, but I noticed that the `FORM` authentication type only accepts passwords when HTTPS is configured. This makes perfect sense from a security standpoint, but it took me a bit of research to understand why the password field was grayed out when only HTTP was configured. It could be helpful to clarify the difference between HTTP and HTTPS for users to avoid confusion.